lib/block_view_test.go

```diff
@@ -125,7 +125,7 @@ func TestBasicTransfer(t *testing.T) {
 		txn.PublicKey = recipientPkBytes
 
 		_signTxn(t, txn, recipientPrivString)
-		utxoView, _ := NewUtxoView(db, params, nil)
+		utxoView, _ := NewUtxoView(db, params, nil, nil)
```

Comment:
It looks like all we need to do to run all of our tests off of Postgres is to do something like:

```
func NewUtxoViewFromEnv(db, pgDB, params) {
  if <env> {
    return NewUtxoView(db, params, nil, nil)
  } else {
    return NewUtxoView(nil, params, nil, pgDB)
  }
}
```

Then in all the places where we initialize the view we can use this function, and we can test PG by running all the tests with a different env. Then eventually we can get rid of all reliance on badger.

Reply:
Probably best to merge what you have now if all tests are passing, but just flagging this as one way we could work the kinks out of PG by utilizing all of our old tests.

Reply:
Yes and yes!